runtime/compiler/control/J9Options.cpp

```diff
@@ -2370,26 +2351,6 @@ bool J9::Options::feLatePostProcess(void * base, TR::OptionSet * optionSet)
       doAOT = false;
       }
 
-   // Determine whether or not to generate method enter and exit hooks
-   //
-   if (!jvmpiInterface(javaVM) || (jvmpiExtensions(jitConfig) & J9JIT_JVMPI_GEN_COMPILED_ENTRY_EXIT))
```

Comment:
You can't fold this away because `jvmpiInterface()` always returns `false`, hence this condition is always `true`.  However, the code below is unconditionally executed now once you do delete `jvmpiInterface()`.

Reply:
Totally agree, my mistake.
Must be fixed now.